src/set.js

```diff
@@ -0,0 +1,103 @@
+'use strict'
+
+export default class Set {
+  constructor(initialValues=[]) {
+    this.elements = initialValues
+  }
+
+  add(value) {
+    if (this.elements.indexOf(value) === -1) {
+      this.elements[this.elements.length] = value
```

Comment:
this is the same as doing `this.elements.push(value)`. You should either just do that or start tracking length on your `Set` instance object and do this `this.elements[this.length] = value`